release/gff/gff.ti.gff_tigrinya/source/gff.ti.gff_tigrinya.model.kps

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package>
+  <System>
+    <KeymanDeveloperVersion>13.0.104.0</KeymanDeveloperVersion>
+    <FileVersion>12.0</FileVersion>
+  </System>
+  <Options>
+    <ExecuteProgram></ExecuteProgram>
+    <ReadMeFile>readme.htm</ReadMeFile>
+    <MSIFileName></MSIFileName>
+    <MSIOptions></MSIOptions>
+  </Options>
+  <StartMenu>
+    <Folder></Folder>
+    <Items/>
+  </StartMenu>
+  <Info>
+    <Name URL="">GFF Tigrinya</Name>
+    <Copyright URL="">© 2020 Ge'ez Frontier Foundation</Copyright>
+    <Author URL="mailto:yacob@geez.org">Ge'ez Frontier Foundation</Author>
+    <Version URL="">1.0</Version>
+  </Info>
+  <Files>
+    <File>
+      <Name>..\build\gff.ti.gff_tigrinya.model.js</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.js</FileType>
+    </File>
+    <File>
+      <Name>welcome.htm</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.htm</FileType>
+    </File>
+    <File>
+      <Name>readme.htm</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.htm</FileType>
+    </File>
+  </Files>
+  <Keyboards/>
+  <LexicalModels>
+    <LexicalModel>
+      <Name></Name>
+      <ID>gff.ti.gff_tigrinya</ID>
+      <Languages>
+        <Language ID="ti-ET">Tigrinya (Ethiopia)</Language>
+        <Language ID="ti-ER">Tigrinya (Eritrea)</Language>

```

Comment:
I'm sorry about the confusing guidance on language tags. (Using `am` to match gff_amharic was correct).

But these keyboards use the tags in the .kps files:

Keyboard | Language Tag | Description
----------- | --------------- | -------------
gff_tigrinya_eritrea | ti-Ethi-ER | Tigrinya (Ethiopic, Eritrea)
gff_tigrinya_ethiopia | ti-Ethi-ET | Tigrinya (Ethiopic, Ethiopia)

The mobile apps will lower-case the language tag and look for exact match to use a lexical-model with a keyboard.

The language tags in .kps and .model_info will need to be consistent. You can leave the directory names as they are

Reply:
Thanks for pointing this out, I understand now.  I'm working on updates to these keyboards this week, to include a touch layout.  When I commit these updates I'll remove the -Ethi , my preference is to not have the -Ethi because it seems redundant, like en-Latn would be.  I'll remove -Ethi in the gff_gurage keyboard as well when I commit its touch layout.

Reply:
The gff_tigrinya_ethiopia keyboard has been updated and includes a touch layout.  It is reset to use the "ti-ET" identifier matching the lexical model.  The keyboard can be downloaded now to a mobile device so is available for testing with the gff_tigrinya lexical model.